database/migrations/2020_01_30_202040_airac_202002_daventry_splits.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+use App\Models\Controller\ControllerPosition;
+use App\Services\AirfieldService;
+use App\Services\HandoffService;
+use App\Services\PrenoteService;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class Airac202002DaventrySplits extends Migration
+{
+    const POSITION_DATA = [
+        [
+            'callsign' => 'LON_CE_CTR',
+            'frequency' => 127.87,
+            'top_down' => [
+                'EGNX' => 'LON_C_CTR',
+                'EGTC' => 'LON_C_CTR',
+                'EGSS' => 'LON_C_CTR',
+                'EGGW' => 'LON_C_CTR',
+                'EGSC' => 'LON_C_CTR',
+            ],
+            'handoff' => [
+                'EGNX_SID_NORTH_09' => 'LON_C_CTR',
```

Comment:
Is this right? Assuming this is LON_C_CTR in absence of NX APC?

Reply:
Sorry - what it's doing is it's effectively saying:

"Insert this position into the handoff order `EGNX_SID_NORTH_09` before `LON_C_CTR`"

Reply:
Oh got it - all good!